internal/vsp/feepayment.go

```diff
@@ -1018,20 +948,9 @@ func (fp *feePayment) confirmPayment() (err error) {
 	return nil
 }
 
-type marshaler struct {
-	marshaled []byte
-	err       error
-}
-
-func (m *marshaler) MarshalJSON() ([]byte, error) {
-	return m.marshaled, m.err
-}
-
-func txMarshaler(tx *wire.MsgTx) json.Marshaler {
+func marshalTx(tx *wire.MsgTx) (string, error) {
 	var buf bytes.Buffer
 	buf.Grow(2 + tx.SerializeSize()*2)
```

Comment:
don't need the extra two bytes in the capacity anymore if we aren't marshaling the quotes

Reply:
Good call. Updated in #decred/vspd/374. Once thats merged I will update this PR.